pyomo/contrib/solver/common/base.py

```diff
@@ -520,20 +556,20 @@ def _solution_handler(
         delete_legacy_soln = True
         if load_solutions:
             if hasattr(model, 'dual') and model.dual.import_enabled():
-                for c, val in results.solution_loader.get_duals().items():
-                    model.dual[c] = val
+                for cons, val in results.solution_loader.get_duals().items():
+                    model.dual[cons] = val
```

Comment:
Style nit (for the future): I would prefer `con` and not `cons` (as I read `cons` as the plural of `con`)